lecture2gether_flask/meta_data_provider.py

```diff
@@ -197,7 +200,15 @@ def get_meta_data(self):
 
         # Get stream url from API
         file_download_url = json.loads(self._drive.files().get_media(fileId=self._file_id).to_json())['uri']
-        self.video_meta_data["streamUrl"] = f"{file_download_url}&l2g_media_type={file_meta['mimeType']}"
+        # Replace backend API key by frontend, so the user can access the video
+        url_parts = urllib.parse.urlparse(file_download_url)
```

Comment:
I think using the API URL from the docu and an f string to add the file id, the key, and the mime type would be a much cleaner and shorter here.